### ISSUE TITLE
Buyer beware: GD1144 badges are being sold as B1248 badges

### ISSUE THREAD

**Issue by Danimations** [Effix/LedBadge#12]
After trawling offerings on eBay and AliExpress, there is definitely some funny business going on re: labeling and marketing of LED badges. Listings often show a unit with a 12 x 48 pixels LED matrix, but then go on to describe a resolution of 11 x 44.

In the case of the first batch of "b1248"s that I bought, they shipped in boxes claiming they were b1248 units. On inspection, they actually had a matrix of 11 x 44 pixels, and sure enough, on inspecting the internals, they were GD1144s (this was spelled out on the PCB).

The GD1144s seem to be flooding the market at present, at the lower cost end of the spectrum (<$15 USD). While this may be good for some (those who just want to scroll text or static images), it doesn't aid us with this project.

Simply put: it appears to be increasingly difficult to buy b1248's with confidence.

I might have to buy some b1236s if things don't work out obtaining genuine b1248s via eBay or AliExpress... but similar dodgy labelling may be at play with those smaller screen badges. 

I really want to put your firmware to work, and as we've discussed, the GD1144 has a totally different circuit and the hardware is not compatible.

**Comment by Effix** [comment]
I imagine they will be harder to find as time goes on. Especially since it makes sense for the manufacturer to cost reduce (while only being outwardly similar).

**Comment by Danimations** [comment]
Some cause for cautious optimism: the eBay seller who sold me the GD1144's labelled as b1248's is going to make amends by sending me another batch of (hopefully) the correct product. It could be 3 weeks or more before they arrive though. How does that mesh with your programming schedule, @Effix ?

**Comment by Effix** [comment]
I'll hopefully have something usable by then.

**Comment by Danimations** [comment]
Excellent! I'll be sure to have some content ready to test the device to capacity. I think I'll focus on 2-bit imagery first.

**Comment by Danimations** [comment]
Using bundled software with 2018-generation badges, I've had to forego a line and four columns of resolution in order to achieve the best available software, badge and playback speed combo. The GD1144 does the job well with images loaded via MiniLED 9.92 software. What it CAN'T do is display 2-bit images (halftones).

**Comment by shawnrancatore** [comment]
@Danimations What USB serial driver works with these GD1144s?  I bought some in Huaqiangbei that were labeled b1248 and they came with a CD with PL2303 drivers, but the USB Vendor ID / Product ID don't match anything like PL2303 and upon internal inspection turn out to have the GD32F150 mcu.
